.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+- React-Native tools version: 
+- VSCode version: 
+- OS version: 
+- React Native version: 
+- Output of the Debug Console (View -> Toogle Debug Console): 
+- Output of the React-Native output channel (View -> Toogle Output -> Select React-Native in ListBox): 
+- Output of the Developer Tools console (Help -> Toggle Developer Tools -> Select Console tab): 
+- Does any other feature seem to not be working as expected?
```

Comment:
Why is this needed? If they have other issues, either create a new issue, or describe below.